website/integrations/services/apache-guacamole/index.md

```diff
@@ -6,45 +6,45 @@ title: Apache Guacamole™
 
 ## What is Apache Guacamole™
 
-> Apache Guacamole is a clientless remote desktop gateway. It supports standard protocols like VNC, RDP, and SSH.
+> Apache Guacamole is a clientless remote desktop gateway which supports standard protocols like VNC, RDP, and SSH.
```

Comment:
```suggestion
> Apache Guacamole is a clientless remote desktop gateway that supports standard protocols like VNC, RDP, and SSH.
```

Reply:
The rule about when to use "that" versus "which" is one most English-speakers don't even know. Use "that" when the following phrase is crucial to the sentence... use "when" if the following phrase is just additional, not-so-important info.